### ISSUE TITLE
Super Mario Kart won't load as low-level emulation

### ISSUE THREAD

**Issue by sharknnth** [byuu/bsnes#155]
Even with the dsp1 and dsp1b data and program roms in the rom folder and bsnes' firmware folder and prefer HLE unchecked, Super Mario Kart always loads as high-level emulation.

**Comment by byuu** [comment]
I'm sorry that it's giving you difficulty, but it does work.
I do acknowledge that ultimately if people are failing to get it to work that I have a usability issue, and I will take that under advisement. Hopefully I can make this easier to use in a future release.

![mariokart](https://user-images.githubusercontent.com/2107894/66534172-59527d00-eb50-11e9-9a16-726c3988ef43.png)

**Comment by jjfmz42015** [comment]
so i tested this.

seems like if hle is checked it shows running in hle if not checked it runs not showing hle but both ways manifest is identical.


Is that proper?

**Comment by Screwtapello** [comment]
I booted SMK with no firmware files around, and it started up with "[HLE]" in the title-bar, signifying no firmware was used.

I placed the `dsp1b.program.rom` and `dsp1b.data.rom` files into `~/.local/share/bsnes/Firmware` (on Linux), restarted bsnes, loaded SMK again, and this time it started up without "[HLE]" in the title-bar, signifying the firmware was found.

> Even with the dsp1 and dsp1b data and program roms in the rom folder

Putting firmware ROMs beside the game you're trying to load doesn't help.

> and bsnes' firmware folder

If you're using a bsnes nightly that comes with an empty "Firmware" directory beside the executable, putting firmware files there should work. Otherwise, putting them in the config-file location (as described above on Linux, `%LocalAppData%\bsnes\Firmware` on Windows) should do the trick.

Make sure you're using the correct files; the SHA256 hashes should be:

```
8546cbac530830446bb8a277f6b139d4ad64d650bdbac7e4e150e2f095665049  dsp1b.data.rom
2eccb54a8f89374911f7e2db48f1b4cde855655e28103f7bda2982a5b418a187  dsp1b.program.rom
```
